fe/fe-core/src/main/java/com/starrocks/analysis/KeysDesc.java

```diff
@@ -151,5 +152,56 @@ public void readFields(DataInput in) throws IOException {
             keysColumnNames.add(Text.readString(in));
         }
     }
+
+    public void checkColumnDefs(List<ColumnDef> cols) {
```

Comment:
This is because the new framework does not support AnalysisException this function use to remove throw AnalysisException define.I will mark @Deprecated to old method.